nft/src/lib.rs

```diff
@@ -306,8 +314,13 @@ pub mod pallet {
 		CannotAddSecretToCapsuleNFTs,
 		/// Operation is not permitted because the NFT is already a secret.
 		CannotAddSecretToSecretNFTs,
-		/// Feature is not available yet
-		ComingSoon,
+		/// Operation is not permitted because the NFT is rented.
+		CannotAddSecretToRentedNFTs,
+		/// Operation is not permitted because the NFT is delegated.
+		CannotAddSecretToDelegatedNFTs,
+		///Enclave which posted the shard for the NFT does not belongs to the
```

Comment:
You are missing an empty space between the forward slash and the start of the first word.